packages/server/src/ChunkExtractor.js

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable react/no-danger */
 import path from 'path'
 import fs from 'fs'
-import uniq from 'lodash/uniq'
-import uniqBy from 'lodash/uniqBy'
-import flatMap from 'lodash/flatMap'
+import uniq from 'lodash/uniq.js'
+import uniqBy from 'lodash/uniqBy.js'
+import flatMap from 'lodash/flatMap.js'
```

Comment:
only external imports need a full specification as everything internal ends up bundled in one file.

The only real problem here - I could not reproduce the problem to verify the fix is "enough"

Reply:
Too many extensions are far less harmful than too few :)